vendor/github.com/gardener/gardener/hack/install-requirements.sh

```diff
@@ -19,7 +19,7 @@ set -e
 echo "> Installing requirements"
```

Comment:
Hmmm, the modes of all the vendored scripts are changed and CI is failing here.
Did you run `make revendor`?

Reply:
Nope forgot it :(